src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -21,6 +21,8 @@ import scala.util.{Success, Try, Failure}
  * Created by dvoet on 10/27/15.
  */
 object UserService {
+  val allUsersGroupRef = RawlsGroupRef(RawlsGroupName("All_Users"))
```

Comment:
Willing to accept that a naughty admin could mess with the group. Can you confirm that trying to create your own group called All_Users results in an HTTP Conflict?